tt1-blocks/block-templates/front-page.html

```diff
@@ -0,0 +1,5 @@
+<!-- wp:template-part {"slug":"header","theme":"tt1-blocks","align":"full", "tagName":"header","className":"site-header"} /-->
```

Comment:
Is this a new file?

Reply:
Wait I see its deleted below... weird.

Reply:
That is weird, I wonder why github didn't catch that it was a file rename.